deploy/components/vllm-sim/deployments.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: vllm
-        image: ghcr.io/llm-d/llm-d-inference-sim:latest
+        image: ${VLLM_SIMULATOR_IMAGE}
```

Comment:
Please undo this change. The file as it was allows the YAML file to be used outside of the kind based tests.